File: VB/CustomizeReportDataExample.Module/BusinessObjects/MyReportDataV2.vb
Imports System.Linq
Imports System.Text
Imports DevExpress.Xpo

Namespace CustomizeReportDataExample.Module.BusinessObjects
    Public Class MyReportDataV2
        Inherits DevExpress.Persistent.BaseImpl.ReportDataV2

        Public Sub New(ByVal session As Session)
            MyBase.New(session)
        End Sub
'INSTANT VB NOTE: The variable category was renamed since Visual Basic does not allow variables and other class members to have the same name:
        Private _category As String
        Public Property Category() As String
            Get
                Return _category
            End Get
            Set(ByVal value As String)
                SetPropertyValue("Category", _category, value)
            End Set
        End Property
    End Class
End Namespace

File: CS/CustomizeReportDataExample.Module/BusinessObjects/MyReportDataV2.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using DevExpress.Xpo;

namespace CustomizeReportDataExample.Module.BusinessObjects {
    public class MyReportDataV2 : DevExpress.Persistent.BaseImpl.ReportDataV2 {
        public MyReportDataV2(Session session) : base(session) { }
        private string category;
        public string Category {
            get { return category; }
            set { SetPropertyValue("Category", ref category, value); }
        }
    }
}
